meta-webserver/recipes-webadmin/cockpit/cockpit_217.bb

```diff
@@ -0,0 +1,178 @@
+SUMMARY = "Admin interface for Linux machines"
+DESCRIPTION = "Cockpit makes it easy to administer your GNU/Linux servers via a web browser"
+
+LICENSE = "LGPLv2+"
+LIC_FILES_CHKSUM = "file://COPYING;md5=4fbd65380cdd255951079008b364516c"
+
+SRC_URI += " \
+    https://github.com/cockpit-project/cockpit/releases/download/${PV}/cockpit-${PV}.tar.xz \
+    file://0001-remove-tests-dep-on-gobject-intro.patch \
+    file://0002-fix-makefile-use-copy-rule-for-unmodified-files.patch \
+    file://0003-feat-add-option-to-build-without-polkit.patch \
+    file://cockpit.pam \
+    "
+SRC_URI[md5sum] = "048fe1f3d9211786b6888e9d2d8db53c"
+SRC_URI[sha256sum] = "6bf41e2ccaebf8b2ede98937750bd33f3d9b369a1de18a1ac17170b59132136c"
+
+inherit gettext pkgconfig autotools systemd distro_features_check
+
+DEPENDS += "glib-2.0-native intltool-native gnutls"
+DEPENDS += "systemd virtual/gettext json-glib krb5 libpam"
```

Comment:
its already inheriting systemd class so DEPENDS on systemd should not be needed.

Reply:
correct

Reply:
Is now corrected